withoutresponses is not a registered marker

Running pytest and pytest-responses with the `--strict` option fails, because "withoutresponses" is not a registered marker. This can be demonstrated with pytest-responses own test suite:

```
$ py.test --strict .
=================================================================== test session starts ====================================================================
platform darwin -- Python 3.5.1, pytest-2.9.2, py-1.4.34, pluggy-0.3.1
rootdir: /Users/keimlink/projects/pytest-responses, inifile: setup.cfg
plugins: responses-0.1.0
collected 0 items / 1 errors

========================================================================== ERRORS ==========================================================================
________________________________________________________ ERROR collecting test_pytest_responses.py _________________________________________________________
test_pytest_responses.py:14: in <module>
    @pytest.mark.withoutresponses
../../.virtualenvs/pytest-responses/lib/python3.5/site-packages/_pytest/mark.py:184: in __getattr__
    self._check(name)
../../.virtualenvs/pytest-responses/lib/python3.5/site-packages/_pytest/mark.py:199: in _check
    raise AttributeError("%r not a registered marker" % (name,))
E   AttributeError: 'withoutresponses' not a registered marker
================================================================= 1 error in 0.02 seconds ==================================================================
```

--- a/pytest_responses.py
+++ b/pytest_responses.py
@@ -6,6 +6,13 @@ import responses
 
 
 # pytest plugin support
+def pytest_configure(config):
+    config.addinivalue_line(
+        'markers',
+        'withoutresponses: Tests which need access to external domains.'
+    )
+
+
 def pytest_runtest_setup(item):
     if not item.get_marker('withoutresponses'):
         responses.start()

--- a/setup.cfg
+++ b/setup.cfg
@@ -1,5 +1,8 @@
 [pytest]
-addopts=--tb=short -p no:doctest
+addopts=
+    --strict
+    --tb=short
+    -p no:doctest
 norecursedirs = bin dist docs htmlcov .* {args}
 
 [bdist_wheel]

--- a/test_pytest_responses.py
+++ b/test_pytest_responses.py
@@ -8,6 +8,8 @@ import pytest
 
 from requests.exceptions import ConnectionError
 
+pytest_plugins = str('pytester')
+
 
 @pytest.mark.withoutresponses
 def test_disabled():
@@ -22,3 +24,8 @@ def test_enabled():
         requests.get('http://responses.invalid')
 
     assert len(responses.calls) == 1
+
+
+def test_marker(testdir):
+    result = testdir.runpytest('--markers')
+    assert '@pytest.mark.withoutresponses' in result.stdout.str()

